zerver/management/commands/makemessages.py

```diff
@@ -29,22 +29,59 @@
 
 """
 
+import os
 import re
+import glob
+import json
```

Comment:
We generally use ujson everywhere in the Zulip backend, since it's faster; maybe it makes sense to use it here as well for consistency?  Obviously not important if there's a reason to use json.